cli/tests/integrations/test_task.py

```diff
@@ -25,12 +25,14 @@
              (SLEEP1, 'test-app1'),
              (SLEEP2, 'test-app2'))
 NUM_TASKS = len(INIT_APPS)
+TASK_ID_COMPLETED = ''
 
 
 def setup_module():
     # create a completed task
     with app(SLEEP_COMPLETED, 'test-app-completed'):
-        pass
+        global TASK_ID_COMPLETED
+        TASK_ID_COMPLETED = _get_completed_task_id()
```

Comment:
You only use `TASK_ID_COMPLETED` in `test_ls_completed`. Can we just move `_get_completed_task_id` in that test?